package.json

```diff
@@ -35,10 +35,12 @@
     "react-scripts": "5.0.1",
     "react-svg": "^15.1.15",
     "react-switch": "^7.0.0",
+    "react-test-renderer": "^18.2.0",
     "react-toastify": "^9.1.1",
     "react-web3-settings": "^1.1.0",
     "redux": "^4.2.0",
     "redux-thunk": "^2.4.2",
+    "save-dev": "^0.0.1-security",
```

Comment:
What is this package for?
The low version number makes me think it's something experimental or not maintained